users/api_views.py

```diff
@@ -0,0 +1,15 @@
+from django.template import Context
+from rest_framework.generics import CreateAPIView
+
+from .models import User
+from .serializers import UserSerializer
+
+
+class UserApiRegistration(CreateAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+
+    def get_serializer_context(self: "UserApiRegistration") -> Context:
```

Comment:
On deeper check this method seems unnecessary. I commented it out and tests pass. Please double-check if it is needed. Chat says it does not.